controllers/rhmi/rhmi_controller.go

```diff
@@ -509,6 +590,13 @@ func (r *RHMIReconciler) handleUninstall(installation *rhmiv1alpha1.RHMI, instal
 		}
 	}
 
+	log.Info("Creating Silence for Keycloak alerts")
```

Comment:
Is it for all alerts?

Reply:
Updated description. Its only one alert that needs to be silenced